app/calendar/calendar-page.tsx

```diff
@@ -0,0 +1,30 @@
+import useSWR from "swr";
```

Comment:
I also moved the logic from page.tsx to this file. The difference is that it receives the username as a parameter